tezos_interop/fetch_storage.js

```diff
@@ -0,0 +1,67 @@
+"use strict";
+
+const fs = require("fs");
+const taquito = require("@taquito/taquito");
+const { RpcClient } = require("@taquito/rpc");
+const { InMemorySigner } = require("@taquito/signer");
+
+const { TezosToolkit } = taquito;
+/**
+ * @typedef Input
+ * @type {object}
+ * @property {string} contract_address
+ * @property {number} confirmation
+ * @property {string} rpc_node
+ */
+/** @returns {Input} */
+const input = () => JSON.parse(fs.readFileSync(process.stdin.fd));
+
+/**
+ * @typedef OutputFinished
+ * @type {object}
+ * @property {"success"} status
+ * @property {rpc.MichelsonV1Expression} storage
+ */
+/**
+ * @typedef OutputError
+ * @type {object}
+ * @property {"error"} status
+ * @property {string} error
+ */
+/** @param {OutputFinished | OutputError} data */
+const output = (data) =>
+  fs.writeFileSync(
+    process.stdout.fd,
+    JSON.stringify({ status: "success", storage: data }, null, 2)
```

Comment:
This is wrong, `error` will call output and the json generated will be `{ status: "success", storage: { status: "error" } }`

Reply:
Sorry about the oversight. Fixed in the next commit.